com.vcvrack.Rack2.yaml

```diff
@@ -0,0 +1,63 @@
+id: com.vcvrack.Rack2
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: rack
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: markdown
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCMAKE_BUILD_TYPE=Release"
+    sources:
+    - type: git
+      url: https://github.com/fletcher/MultiMarkdown-6.git
+      tag: 6.7.0
+
+  - name: rack2free
+    buildsystem: simple
+    build-options:
+      build-args:
+        - --share=network
```

Comment:
build-args has no effect on Flathub, Build is done offline. https://docs.flathub.org/docs/for-app-authors/requirements#no-network-access-during-build

Reply:
Oh hell, I'm giving up here as I don't see any reason to reinvent wheels and mimic all Makefiles recursively. That's why I was insisting on using prebuilt package from developer's site. I'm closing PR and happily use my precious local bundle.